ios/flutter_barcode_scanner.podspec

```diff
@@ -14,6 +14,7 @@ A new Flutter plugin supports barcode scanning on both Android and iOS.
   s.source           = { :path => '.' }
   s.source_files = 'Classes/**/*.{swift,h,m}'
   s.public_header_files = 'Classes/**/*.h'
+  s.resources = 'Assets/*.png'
```

Comment:
This line is important. Those image assets are not bundled with the plugin without this one.